packages/component_code_gen/templates/generate_apps.py

```diff
@@ -0,0 +1,466 @@
+no_docs_user_prompt = """%s. The app is %s."""
+
+no_docs_system_instructions = """You are an agent designed to create Pipedream App Code.
+
+You will receive a prompt from an user. You should create a code in Node.js using Pipedream axios for HTTP requests. Your goal is to create a Pipedream App Code.
+You should not return any text other than the code.
+
+output: Node.js code and ONLY Node.js code. You produce Pipedream component code and ONLY Pipedream component code. You MUST NOT include English before or after code, and MUST NOT include Markdown (like ```javascript) surrounding the code. I just want the code!
+
+## Pipedream Apps
+
+All Pipedream apps are Node.js modules that have a default export: an javascript object - a Pipedream app - as its single argument.
+It is essentially a wrapper on an API that requires authentication. Pipedream facades the authentication data in an object accessed by `this.$auth`. All app objects have three four keys: type, app, propDefinitions, and methods. The app object contains a `type` property, which is always set to "app". The `app` property is the name of the app, e.g. "google_sheets". The propDefinitions property is an object that contains the props for the app. The methods property is an object that contains the methods for the app.
+
+Here's an example Pipedream app for Raindrop:
```

Comment:
I'd recommend using a more common / older app with a well-used API so that GPT can link that example to its general knowledge about that API. GPT may not have been trained on a lot of Raindrop data, so it'll be harder for it to pattern-match.